ppr-ui/src/utils/config-helper.ts

```diff
@@ -136,4 +136,9 @@ export async function fetchConfig (): Promise<any> {
     (<any>window).addressCompleteKey = addressCompleteKey
     console.info('Set Address Complete Key.')
   }
+
+  const siteminderLogoutUrl: string = response.data.SITEMINDER_LOGOUT_URL
+  if (siteminderLogoutUrl) {
+    sessionStorage.setItem('SITEMINDER_LOGOUT_URL', siteminderLogoutUrl)
```

Comment:
Is this used anywhere?

Reply:
I was curious about this as well. 
Gary mentioned this was intentional as it's a dependency of recent changes in the SBC Common Components, although i'm not sure the details.